server/devtool/webeditor/pom.xml

```diff
@@ -183,7 +177,39 @@
 			<artifactId>projectrepository-file</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.eclipse.vorto</groupId>
+			<artifactId>projectrepository-modeshape</artifactId>
+			<version>${project.version}</version>
+		</dependency>
 
+		<!--Java Content Repository API -->
+		<dependency>
+			<groupId>javax.jcr</groupId>
+			<artifactId>jcr</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.modeshape</groupId>
+			<artifactId>modeshape-sequencer-xml</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.modeshape</groupId>
+			<artifactId>modeshape-jcr</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.modeshape</groupId>
+			<artifactId>modeshape-jcr-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.modeshape</groupId>
+			<artifactId>modeshape-common</artifactId>
+		</dependency>
+        <dependency>
+            <groupId>org.modeshape</groupId>
```

Comment:
Is this dependency to webdav required ?

Reply:
No, that is not need
Added that during testing, forgot to remove that